demisto_sdk/commands/common/update_id_set.py

```diff
@@ -389,7 +414,9 @@ def get_integration_data(file_path):
     integration_api_modules = get_integration_api_modules(file_path, data_dictionary, is_unified_integration)
     default_classifier = data_dictionary.get('defaultclassifier')
     default_incident_type = data_dictionary.get('defaultIncidentType')
+    is_fetch = data_dictionary.get('script', {}).get('isfetch', False)
```

Comment:
Why did we add this?

Reply:
It was requested several times in the last few days.
It's unrelated to the issue (I can move to another PR).